src/views/canvas/CanvasSeqBlock.js

```diff
@@ -117,6 +117,7 @@ const View = boneView.extend({
 
     events.mousewheel = "_onmousewheel";
     events.DOMMouseScroll = "_onmousewheel";
+    events.wheel = "_onmousewheel";
```

Comment:
The other scroll handlers at the top don't work anymore in FireFox.
see https://developer.mozilla.org/en-US/docs/Web/API/Element/mousewheel_event and https://developer.mozilla.org/en-US/docs/Web/API/Element/DOMMouseScroll_event

`wheel` is the new standard, and this change makes wheel scrolling work on FireFox.